README.md

```diff
@@ -1,51 +1,9 @@
-# v2 - wip
+# Eslint configs / plugins monorepo
 
-# @bigcommerce/eslint-config
+For installation guide and setup please visit each package's readme.
 
-This package is a configuration preset for [ESLint](https://eslint.org/).
+## Packages
 
+[@bigcommerce/eslint-config](packages/eslint-config)
 
-## Install
-
-```sh
-npm install --save-dev eslint prettier
-npm install --save-dev @bigcommerce/eslint-config
-```
-
-
-## Usage
-
-Add `@bigcommerce/eslint-config` to your project's ESLint configuration file. i.e.:
-
-```js
-// .eslintrc.js
-require('@bigcommerce/eslint-config/patch');
-
-module.exports = {
-  extends: ['@bigcommerce/eslint-config'],
-};
-```
-
-This config also runs prettier via eslint, add the following to your `package.json`
-```json
-{
-  "prettier": "@bigcommerce/eslint-config/prettier"
-}
-```
-
-If possible, try not to override the preset unless you have a special reason.
-
-## Release
-
-To release:
-
-```sh
-npm run release
-```
-
-Please refer to the documentation of [standard-version](https://github.com/conventional-changelog/standard-version) for more options.
-
-
-## License
-
-MIT
+[@bigcommerce/eslint-plugin](packages/eslint-plugin)
```

Comment:
Should we discourage the use of the plugin? maybe just omit it or make the main readme point to the one in `eslint-config`? 

what do u think?